Sources/ClientRuntime/Networking/Http/HttpResponse.swift

```diff
@@ -23,7 +35,7 @@ public class HttpResponse: HttpUrlResponse {
     }
 }
 
-extension HttpResponse: CustomDebugStringConvertible {
+extension HttpResponse {
```

Comment:
Presuming this protocol was removed because it requires a non-async method or var.

Is the rest of this extension of any use anymore?

Reply:
Yep the protocol conformance was removed because the computed property `debugDescription` is now actor-isolated and can't fulfill the protocol requirement. But the properties are still used in other places so I left the extension as is.